Source/SwiftLintFramework/Rules/Lint/CommentSpacingRule.swift

```diff
@@ -122,15 +122,8 @@ struct CommentSpacingRule: SourceKitFreeRule, ConfigurationProviderRule, Substit
     func violationRanges(in file: SwiftLintFile) -> [NSRange] {
         // Find all comment tokens in the file and regex search them for violations
         file.syntaxClassifications
-            .compactMap { (classifiedRange: SyntaxClassifiedRange) -> [NSRange]? in
-                switch classifiedRange.kind {
-                case .blockComment, .docBlockComment, .lineComment, .docLineComment:
-                    break
-                default:
-                    return nil
-                }
-
-                let range = classifiedRange.range.toSourceKittenByteRange()
+            .map { $0.range.toSourceKittenByteRange() }
```

Comment:
The `isComment` seems to be missing here.

Reply:
That's why I get for doing manual copy&paste.
Thank you so much for the thorough review!

I'm kinda surprised by the lack of false positives with this mistake. I mean, the list of examples is not exhaustive but the rule was also run on all those other repos right?